tests/tools/internal-rules/no-invalid-meta.js

```diff
@@ -126,6 +126,24 @@ ruleTester.run("no-invalid-meta", rule, {
                 column: 18
             }]
         },
+        {
+            code: [
+                "module.exports = {",
+                "    meta: [],",
```

Comment:
meta could be an object. is the `[]` intentional?

Reply:
Yes. In case of an emtpy object, `node.properties` is `[]` but for any other type it is `undefined` resulting in a crash. I intentionally declared `meta` as an `array` to elicit this case

Reply:
I see.thanks for the clarification!